src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -2,12 +2,16 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 // This program uses code hyperlinks available as part of the HyperAddin Visual Studio plug-in.
 // It is available from http://www.codeplex.com/hyperAddin 
+#if !PLATFORM_UNIX
+
 #define FEATURE_MANAGED_ETW
```

Comment:
Vance, I'm assuming this file is included in other projects and that's why these defines exist in addition to the settings coming from the .props file.  Let me know if that's not the case and these should just be deleted.